RepoData.json

```diff
@@ -0,0 +1,124 @@
+        "System.Net.Requests": [ "4.0.0", "3.9.0" ],
+        "System.Net.Sockets": "4.1.0",
+        "System.Net.TestData": "1.0.0-prerelease",        
+        "System.Net.WebHeaderCollection": "4.0.0",
+        "System.Net.WebSockets": "4.0.0",
+        "System.Numerics.Vectors": [ "4.1.0", "4.0.0" ],
+        "System.ObjectModel": [ "4.0.10", "4.0.12", "4.0.0" ],
+        "System.Private.Uri": "4.0.0",
+        "System.Reflection": [ "4.0.10", "4.0.0", "4.1.0" ],
+        "System.Reflection.Emit": "4.0.0",
+        "System.Reflection.Emit.ILGeneration": "4.0.0",
+        "System.Reflection.Emit.Lightweight": "4.0.0",
+        "System.Reflection.Extensions": [ "4.0.1", "4.0.0" ],
+        "System.Reflection.Primitives": "4.0.0",
+        "System.Reflection.TypeExtensions": "4.0.0",
+        "System.Resources.ResourceManager": [ "4.0.0", "4.0.1" ],
+        "System.Runtime": [ "4.0.0", "4.0.20", "4.0.10", "4.1.0" ],
+        "System.Runtime.Extensions": [ "4.0.10", "4.0.0", "4.1.0" ],
+        "System.Runtime.Handles": [ "4.0.0", "4.0.1" ],
+        "System.Runtime.InteropServices": [ "4.0.20", "4.0.0", "4.0.10", "4.1.0" ],
+        "System.Runtime.Numerics": "4.0.0",
+        "System.Runtime.Serialization.Primitives": [ "4.0.0", "4.1.1" ],
+        "System.Runtime.Serialization.Xml": "4.0.0",
+        "System.Runtime.WindowsRuntime": [ "4.0.0", "4.0.10" ],
+        "System.Security.AccessControl": "4.0.0",
+        "System.Security.Claims": "4.0.0",
+        "System.Security.Cryptography.Algorithms": "4.2.0",
+        "System.Security.Cryptography.Cng": "4.2.0",
+        "System.Security.Cryptography.Encoding": "4.0.0",
+        "System.Security.Cryptography.Primitives": "4.0.0",
+        "System.Security.Cryptography.X509Certificates": "4.1.0",
+        "System.Security.Cryptography.X509Certificates.TestData": "1.0.2-prerelease",        
+        "System.Security.Principal": "4.0.0",
+        "System.Security.Principal.Windows": "4.0.0",
+        "System.Text.Encoding": [ "4.0.10", "4.0.0" ],
+        "System.Text.Encoding.Extensions": [ "4.0.10", "4.0.0" ],
+        "System.Text.RegularExpressions": [ "4.0.0", "4.0.10" ],
+        "System.Threading": ["4.0.0", "4.0.10", "4.0.11"],
+        "System.Threading.Overlapped": "4.0.0",
+        "System.Threading.Tasks": [ "4.0.10", "4.0.0" ],
+        "System.Threading.Tasks.Parallel": "4.0.0",
+        "System.Threading.Thread": "4.0.0",
+        "System.Threading.ThreadPool": "4.0.10",
+        "System.Xml.ReaderWriter": [ "4.0.0", "4.0.10" ],
+        "System.Xml.XDocument": "4.0.0",
+        "System.Xml.XmlDocument": "4.0.0",
+        "System.Xml.XmlSerializer": "4.0.0",
+        "System.Xml.XPath": "4.0.0",
+        "xunit": "2.1.0",
+        "xunit.console.netcore": "1.0.3-prerelease-00607-01",
+        "xunit.runner.console": "2.1.0",
+        "xunit.runner.utility": "2.1.0"        
+    },
+
+    "toolsetPackages": [
+        "Microsoft.DotNet.BuildTools",
```

Comment:
Build Dependencies: artifacts which are referenced by the build output and include NuGet packages, MSI, etc ... In order to support a composed build these are further divided into floating and static dependencies: ◦Floating: dependencies where versions can change as a part of a composed build via the  change  command. This commonly includes CoreFx, CoreClr, etc ...

◦Static: dependencies whose version do not change via the  change  command. This is commonly used for SDK binaries, legacy tools, etc ...


•Toolset Dependencies: artifacts used in the production of build output. This commonly includes NuGet.exe, compilers, etc ...